pages/linux/rig.md

```diff
@@ -0,0 +1,24 @@
+# rig
+
+> Utility that will piece together a random first name, last name, street number and address, along with a geographically consistant (ie, they all match the same area) city, state, ZIP code, and area code.
+> More information: <http://manpages.ubuntu.com/manpages/bionic/man6/rig.6.html>.
+
+- Return a random name and address:
+
+`rig`
+
+- Specify male/female name. If neither or both options are selected, RIG uses both female and male names:
```

Comment:
If we move the randomly gender to the previous example, we can say this shorter:
```suggestion
- Display a [m]ale (or [f]emale) random name and address:
```
Or we can replace this with 2 examples, 1 for male, 1 for female.

Reply:
@neel24 what about this?